src/main/java/sx/blah/discord/api/internal/DiscordVoiceWS.java

```diff
@@ -280,8 +282,7 @@ private void setupReceiveThread() {
 					Discord4J.LOGGER.error(LogMarkers.VOICE_WEBSOCKET, "Discord Internal Exception", e);
 				}
 			}
-		};
-		executorService.scheduleAtFixedRate(receiveThread, 0, OpusUtil.OPUS_FRAME_TIME_AMOUNT, TimeUnit.MILLISECONDS);
```

Comment:
The 20ms pause was purposeful. See: https://github.com/austinv11/Discord4J/blob/audio-receive/src/main/java/sx/blah/discord/handle/audio/IAudioReceiver.java#L17 it is so that data is consistent in length when handled by receivers